forte/data/extractor/char_extractor.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Missing the module docstring

Reply:
I am not sure what is module docstring. Can you give me an example?

Reply:
Normally we need a docstring to be added on all modules, like this:

https://github.com/asyml/forte/blob/master/forte/processors/eliza_processor.py#L14

However, I just realize many of our files are missing this. I probably forget to enforce this earlier. But we'd better start doing this now.